locksmith/src/controllers/metadataController.ts

```diff
@@ -10,11 +10,11 @@ const config = require('../../config/config')[env]
 const metadataOperations = require('../operations/metadataOperations')
 
 namespace MetadataController {
-  export const data = async (req: Request, res: Response): Promise<any> => {
+  export const data = async (req: any, res: Response): Promise<any> => {
```

Comment:
Why did this type signature change?

Reply:
Previously we were using the standard Request type here, in the new filter that this sits in front of this i've add a `signee` if there is one.

It might be worthwhile to make a type for this in the future but i'd rather let it soak a bit to see if anything changes after everything is in place